node/hyperbahn/handler.js

```diff
@@ -178,9 +195,20 @@ function sendAdvertise(services, options, callback) {
 
     res: {}
 */
+
 HyperbahnHandler.prototype.handleRelayAdvertise =
 function handleRelayAdvertise(self, req, arg2, arg3, cb) {
-    /*eslint max-params: [2, 5]*/
+    self.handleRelay(req, arg2, arg3, cb, self.advertise);
```

Comment:
self.advertise.bind(self) is a correct way, but @Raynos said it is not efficient.

Another way is to pass in the endpoint name and do an if-else in self.handleRelay